vxyowsup/tests/test_whatsapp.py

```diff
@@ -39,15 +46,28 @@ def setUp(self):
         }
 
         self.transport = yield self.tx_helper.get_transport(self.config)
+        self.testing_layer = self.transport.stack_client.network_layer
 
-    def test_nothing(self):
-        pass
+    def assert_nodes_equal(self, node1, node2):
+        node2["id"] = node1["id"]
```

Comment:
Ideally we should somehow arrange for the node created to have an id that we can explicitly test but I'm not 100% sure how to do that yet. Perhaps just add a "TODO: ..." here so we don't forget to fix it later?